### PR TITLE
Fixed navbar styling.

### PR DESCRIPTION
**Pull request by stewartTS**
https://trello.com/c/pFqUXpFH/39-fix-navbar-styling

### CODE REVIEW

File: WhaleSpotting/react-app/src/styles/Constants.scss [hunk 1]
```diff
@@ -38,7 +38,7 @@ $guttersSide: 20px;
 $minMobileScreenSize: 320px;
 $mobileScreenSize: 445px;   
 
-$headerHeight: 6vh;
+$headerHeight: 40px;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Constants.scss]: So if we are changing this to px, we should also probably change the footer and mobile styling. I am not sure this needs to be px, though, does this make the styling nicer?

File: WhaleSpotting/react-app/src/styles/Profile.scss [hunk 2]
```diff
@@ -8,7 +8,8 @@
 }
 
 .profile-pane {
-    min-height: 100vh - $footerHeight - $headerHeight;
+    min-height: 100%;
+    box-sizing: border-box;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Profile.scss]: I'm not sure this works as you want, instead you can do 
```suggestion
    min-height: calc(100vh - #{$footerHeight} - #{$headerHeight});
```

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/styles/Profile.scss (hunk 2 above)
Comment (stewartTS) [reply WhaleSpotting/react-app/src/styles/Profile.scss]: Added

File: WhaleSpotting/react-app/src/styles/Constants.scss (hunk 1 above)
Comment (stewartTS) [reply WhaleSpotting/react-app/src/styles/Constants.scss]: I personally believe so, it makes everything more precise.

**Review by oskarwilliams — CHANGES_REQUESTED** [review]
Change header height

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**